src/Analyzers/Core/Analyzers/Helpers/AnalyzerOptionsExtensions.cs

```diff
@@ -6,13 +6,27 @@
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.CodeCleanup;
+using Microsoft.CodeAnalysis.Formatting;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
 
 #if CODE_STYLE
 using TOption = Microsoft.CodeAnalysis.Options.IOption2;
 #else
 using TOption = Microsoft.CodeAnalysis.Options.IOption;
 #endif
 
+#if CODE_STYLE
+namespace Microsoft.CodeAnalysis.CodeCleanup
+{
+    internal readonly struct CodeCleanupOptions
+    {
+    }
```

Comment:
is this the general pattern in CODE_STYLE?  just an empty struct since there is nothing to fallback to?  just trying to figure this out.

Reply:
yeah, i prefer to stub out as needed instead of having to #ifdef all use sites.